CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Maintainers
+This repo is currently maintained by Ken Liao (@yknl). 
+
+# Pull Request Reviewers
```

Comment:
Better organization.

`# Pull Requests`

`## Submitting`

`## Reviewers and Reviewing`